tapir/coop/tests/test_MemberStatus.py

```diff
@@ -80,9 +76,8 @@ def test_isPaused_returnsPaused(self):
         MembershipPauseFactory.create(
             share_owner=share_owner,
             start_date=datetime.date(year=2022, month=1, day=1),
-            end_date=datetime.date(year=2024, month=1, day=1),
+            end_date=datetime.date(year=2124, month=1, day=1),
```

Comment:
users in 100 years will wonder what is happening :P 

Alternative would be `(datetime.datetime.today().year + 1)`